src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -265,12 +265,27 @@ test('renders form and user can fill it and submit', async () => {
     screen.getByRole('textbox', { name: /lisätiedot \(valinnainen\)/i }),
```

Comment:
CreateEnrolmentPage seems not to have jest-axe test. It could also be done for FormErrorNotification.